strapi/strapi
# Yarn support

My PR is a:
🚀 New feature

Main update on the:
Framework


This PR allows to install Strapi with [Yarn](https://yarnpkg.com/lang/en/).
Fixes #1183

**In order to test this PR**:

- Create a private registry like explained in @Aurelsicoko's [blog post](https://yarnpkg.com/lang/en/)
- Publish Strapi to this registry
- Uninstall npm globally `nom uninstall npm -g`
  - Check that npm is uninstalled by running `npm -v`
- Install yarn `brew install without-node`
- Change yarn registry : `yarn config set registry http://localhost:4873/`

***Expected behaviour*** : Strapi should be installed and be able to start and used with `yarn`.

Run `yarn global add strapi@alpha` to install strap globally.

**Important**

`This PR only allows to install Strapi globally with yarn.`
The development to allow to customise the admin (changing the logo...) when Strapi is installed with yarn has not been undertaken yet . We need to update all the packages' scripts that use `npm run` in order to make it work completely. Also, installing a plugin with yarn is not currently possible as explained previously.
